ObjectMemory_Constants.h

```diff
@@ -26,6 +26,33 @@
  * A size that is too big to appear in one of the small free chunk lists.
  */
 #define BigSize 0x2e
+/**
+ * A run of object pointers that can be used to hold the free memory when the VM starts.
+ */
+#define FirstFreeObject 0x7F00
```

Comment:
As the comment says, these object pointers are used when the VM starts. They will quickly get re-written when the allocator fragments their memory, and eventually when the marking collector runs they'll lose all meaning. But on startup we need to make sure that they hold a reference to the free memory on the heap, and aren't marked as in-use.